jsonParser.h

```diff
@@ -1,3 +1,14 @@
+/**
+ * \class jsonParser
+ * 
+ * \brief jsonParser class
+ * 
+ * Here we do the parse from a json input.
+ * 
+ * \date 2020/10/27
+ * 
+*/
+
```

Comment:
A .h fileokban a class részhez mindenhova be lehetne írni az author és version tagokat is.
Ezt leszámítva, teljesen korrektek a leírások.👌🔥